packages/ui/src/DateTimeField.tsx

```diff
@@ -108,9 +108,9 @@ export const DateTimeField = ({
         const month = cleanedInput.slice(0, 2);
         const day = cleanedInput.slice(2, 4);
         const year = cleanedInput.slice(4, 8);
-        parsedDate = DateTime.fromFormat(`${month}${day}${year}`, "MMddyyyy", {
-          zone: timezone,
-        });
+        parsedDate = DateTime.fromFormat(`${month}${day}${year}`, "MMddyyyy", {zone: timezone})
```

Comment:
It's working correctly as written and not at all urgent, but if you are selecting the time in a different time zone than your local, when it goes back to the field, it feels to me like the field should show what you selected rather than converting it back to your local. So if i am scheduling and i select eastern, when i go back, the field is showing the central time still. Is that what we want? Doesn't totally feel intuitive to me so it feels worth noting, but i'm good to keep it this way if it makes more sense to everyone else.

https://github.com/user-attachments/assets/c8dab2ff-9bd0-4db3-b6c9-8c8622046f29